apps/ledger-live-mobile/src/components/WebPlatformPlayer/WebViewV2.tsx

```diff
@@ -412,6 +421,97 @@ export const WebView = ({ manifest, inputs }: Props) => {
           );
         },
       );
+
+      serverRef.current.setHandler("device.transport", ({ appName }) => {
+        if (transport.current) {
+          return Promise.reject(new Error("Device already opened"));
+        }
+
+        tracking.deviceTransportRequested(manifest);
+
+        return new Promise((resolve, reject) => {
+          navigation.navigate(ScreenName.DeviceConnect, {
+            appName,
+            onSuccess: result => {
+              tracking.deviceTransportSuccess(manifest);
+              if (!result.device) {
+                reject(new Error("No device"));
+                return;
+              }
+              const { deviceId } = result.device;
+
+              transport.current = getTransport({
+                deviceId,
+              });
+
+              // Clean the ref on completion
+              transport.current.subscribe({
+                complete: () => {
+                  transport.current = undefined;
+                },
+              });
+              resolve("1");
```

Comment:
Why `"1"`?

Reply:
This acts as the `transportId` for ledger live, we only use `"1"` here because ledger-live will only handle one transport at a time (this can change later one)

I still need to rename the `deviceId` to `transportId`